airbyte-webapp/package.json

```diff
@@ -3,17 +3,18 @@
   "version": "0.40.32",
   "private": true,
   "engines": {
-    "node": "16.18.1"
+    "node": "16.18.1",
+    "pnpm": "7.26.2"
   },
   "scripts": {
     "prepare": "cd .. && husky install airbyte-webapp/.husky",
-    "prestart": "npm run generate-client",
-    "start": "node -r ./scripts/dev-overwrites.js ./node_modules/.bin/vite",
-    "prestart:cloud": "npm run generate-client",
-    "start:cloud": "AB_ENV=${AB_ENV-frontend-dev} node -r ./scripts/environment.js -r ./scripts/dev-overwrites.js ./node_modules/.bin/vite",
-    "prebuild": "npm run generate-client",
+    "prestart": "pnpm run generate-client",
+    "start": "NODE_OPTIONS='-r ./scripts/dev-overwrites.js' vite",
```

Comment:
We should update the airbyte-webapp/README.md, since it still contains `npm` commands

Reply:
Very good call. I've actually rewritten the README now completely to have less outdated information and just point to our official documentation for development information, so we don't have this duplicated across different places.